.github/workflows/release-generate-ci.yaml

```diff
@@ -230,246 +237,6 @@ jobs:
           fi
           gh pr create --base "$target_branch" --head "serverless-qe:$branch" --title "[$target_branch] Update dependabot configurations" --body "Update dependabot configurations" || true
         working-directory: ./src/github.com/openshift-knative/hack/openshift-knative/backstage-plugins
-      - env:
-          GH_TOKEN: ${{ secrets.SERVERLESS_QE_ROBOT }}
```

Comment:
why is all of this removed? I think this was only for testing
btw: You can run `make generate-ci-action` to update this file too

Reply:
should be fixed now